src/NServiceBus.Transport.RabbitMQ/Administration/ManagementClient/Models/FeatureFlags.cs

```diff
@@ -0,0 +1,32 @@
+﻿#nullable enable
+
+namespace NServiceBus.Transport.RabbitMQ.Administration.ManagementClient.Models;
+
+static class FeatureFlags
+{
+    /// <summary>
```

Comment:
We typically don't put XML comments on non-public types since that's mainly for intellisense for consuming projects. If there's not a compelling reason to have them here, I'd prefer to stay consistent and remove them.